code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -294,3 +294,9 @@ Des: Removes all infected images from the alien.
 				playsound(T, S, volume, 1, range)
 				return 1
 	return 0
+
+/mob/living/carbon/alien/getTrail()
+	if(getBruteLoss() < 150)

```

Comment:
`200`

if it cause problems, make sure you have the sprites for both `xltrails` and `xttrails`